test/_unit.js

```diff
@@ -479,6 +479,8 @@ suite('Lexer', () => {
     const { exports } = parse(`
       module.exports = { 'ab cd': foo };
       exports['not identifier'] = 'asdf';
+      exports['\u{D83C}\u{DF10}'] = 1;
```

Comment:
I think you either need to use `\\`, or `String.raw`. Otherwise the code doesn't contain the `\u{D83C}` substring, but the 0xD83C character.

Reply:
Good catch thanks.